### PR TITLE
Added integrating with Sumo Logic topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on https://help.stackrox.com/docs/integrate-with-other-tools/integrate-with-sumologic/

Preview: https://openshift-docs-git-sumologic-gnelson.vercel.app/openshift-acs/master/integration/integrate-with-sumologic.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
This PR uses a module from https://github.com/openshift/openshift-docs/pull/34022 Please review this after #34022

File: integration/integrate-with-sumologic.adoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+:context: integrate-with-sumologic
+
+toc::[]
+
+[role="_abstract"]
+If you are using https://www.sumologic.com/[Sumo Logic], you can forward alerts from {product-title} to Sumo Logic.
+
+The following steps represent a high-level workflow for integrating {product-title} with Sumo Logic:
+
+. Add a new Custom App in Sumo Logic, set HTTP source, and get the HTTP URL.
```
Comment (bobfuru) [inline integration/integrate-with-sumologic.adoc:13]: "set the HTTP source"

File: modules/configure-sumologic.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+= Configuring Sumo Logic
+
+Use the *Setup Wizard* to set up *Streaming Data* and get the HTTP URL.
+
+.Procedure
+. Log in to your Sumo Logic Home page and select *Setup Wizard*.
+. Move your cursor over to *Set Up Streaming Data* and select *Get Started*.
+. On the Select Data Type page, select *Your Custom App*.
+. On the Set Up Collection page, select *HTTP Source*.
+. Enter a name for *Source Category*, for example, `stackrox` and click *Continue*.
```
Comment (bobfuru) [inline modules/configure-sumologic.adoc:15]: s/stackrox/rhacs/ ?

File: modules/sumologic-configuring-acs.adoc [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+:_module-type: PROCEDURE
+[id="sumologic-configuring-acs_{context}"]
+= Configuring {product-title}
+
+Create a new integration in {product-title} by using the HTTP URL.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Scroll down to the *Notifier Integrations* section and select *Sumo Logic*.
+. Click *New Integration* (add icon).
```
Comment (bobfuru) [inline modules/sumologic-configuring-acs.adoc:13]: I think you want to markup as `(`**add**` icon)` and similar for the rest of the PR.

File: modules/view-alerts-in-sumo-logic.adoc [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+//
+// * integration/integrate-with-sumologic.adoc
+:_module-type: PROCEDURE
+[id="view-alerts-in-sumo-logic_{context}"]
+= Viewing alerts in Sumo Logic
+
+You can view alerts from {product-title} in Sumo Logic.
+
+. Log in to your Sumo Logic Home page and click *Log Search*.
+. In the search box, enter `_sourceCategory=stackrox`.
```
Comment (bobfuru) [inline modules/view-alerts-in-sumo-logic.adoc:11]: Also confirming that `stackrox` is the right parameter here.

**Review by bobfuru — COMMENTED** [review]
A few tiny suggestions/clarifications and otherwise LGTM!

**Comment by gaurav-nelson** [comment]
Thank you @bobfuru fixed, rebased and ready to merge.

**Review by bobfuru — COMMENTED** [review]
LGTM

**Pull request merged**